src/lib.rs

```diff
@@ -1077,7 +1080,7 @@ pub fn is_all_exists<K: AsRef<OsStr>>(keys: &Vec<K>) -> bool {
 ///     assert!(output.is_ok());
 /// }
 /// ```
-pub fn load_file(file: &str) -> Result<(), EnvmntError> {
+pub fn load_file(file: &str) -> EnvmntResult<()> {
```

Comment:
i saw this change everywhere. this is great!!! not sure why i didn't do it :)

Reply:
“Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away.” ~Antoine de Saint-Exupéry

Mind you, I'd never be as arrogant as to presume I'm even close to grasp the concept of perfection when it comes to programming. Oh boy, or if it is even remotely possible for our limited brains...